docs/self-managed/setup/install.md

```diff
@@ -104,6 +155,8 @@ Installing all the components in a cluster requires all Docker images to be down
 
 For air-gapped environments, refer to [installing in an air-gapped environment](/self-managed/setup/guides/air-gapped-installation.md).
 
+This command does not install Identity.
```

Comment:
I feel like this is not accurate.  Identity by default should be enabled.

Did you try going through this guide, and found that the identity pod is not getting created? if so, that might be a bug.

Reply:
You are right. It was an artifact from my first iteration.